sdk/core/azure-core/src/main/java/com/azure/core/credential/AccessToken.java

```diff
@@ -64,6 +82,17 @@ public OffsetDateTime getExpiresAt() {
         return expiresAt;
     }
 
+    /**
+     * Gets the time when the token should refresh, in UTC.
+     *
+     * Note: This value can be null.
+     *
+     * @return The time when the token should refresh, in UTC.
+     */
+    public OffsetDateTime getRefreshOn() {
```

Comment:
Should we consider naming this `getNextRefreshAt()` to indicate this is the expected time for next token refresh and not the last refresh time? Also, since we are using `expiresAt()`, we should use `At` here as well.

Reply:
We should use `At` instead of `On` to be consistent within the same class as we already have `getExpiresAt()`. We can drop `Next` from the name if we can clearly document it.